### DISCUSSION TITLE
Port 53

### DISCUSSION THREAD

**Discussion by EmptyWolf** [AdguardTeam/AdGuardHome#5314] [Q&A]
Hi everyone, 

Firstly, happy new year. 

Secondly, a question about ports used by AdGuard and specifically port 53. I am currently running AdGuard Home from a Docker container on my Synology NAS. However, I recently wanted to install the DNS Server package on the NAS and it seems to crash and require a "repair" (which does not work) due to port 53 being occupied by AdGuard. I have contacted Synology Support who told me port 53 is absolutely necessary for DNS Server to run. Therefore, I'm now looking on the other side of the fence to check if AdGuard absolutely needs port 53 or if this can be modified in some way so that I can run both of them (AdGuard and DNS Server) simultaneously (or if I have make an "either / or" decision).

Sorry if this is an obvious (or blatantly stupid) question, I am kind of new to this and my research has been unfruitful so far.

Thank you.
MT

**Comment by ammnt** [comment]
> Hi everyone,
> 
> Firstly, happy new year.
> 
> Secondly, a question about ports used by AdGuard and specifically port 53. I am currently running AdGuard Home from a Docker container on my Synology NAS. However, I recently wanted to install the DNS Server package on the NAS and it seems to crash and require a "repair" (which does not work) due to port 53 being occupied by AdGuard. I have contacted Synology Support who told me port 53 is absolutely necessary for DNS Server to run. Therefore, I'm now looking on the other side of the fence to check if AdGuard absolutely needs port 53 or if this can be modified in some way so that I can run both of them (AdGuard and DNS Server) simultaneously (or if I have make an "either / or" decision).
> 
> Sorry if this is an obvious (or blatantly stupid) question, I am kind of new to this and my research has been unfruitful so far.
> 
> Thank you. MT

You can try to bind another port for plain DNS instead 53 via AdGuardHome.yaml config file:
```
dns:
  bind_hosts:
    - 127.0.0.1
  port: 5353
```
But you also need to change or add the port in client's settings🙂

**Reply by EmptyWolf** [reply]
Hello ammnt, 

Thanks for your reply. 

However, I am still a bit stuck (unfortunately). When opening the Docker container from which AdGuard is running in the Synology interface, I can indeed see the file you are referring to (AdGuardHome.yaml).
![image](https://user-images.githubusercontent.com/92716596/212500079-544da347-b19f-40c7-84c2-aab942339815.png)
That said, this file is not accessible to me via File Station. So I tried to reach it by SSH-ing into the NAS but I cannot go any further than /opt for some reason. Any time I try any command here, the terminal returns "containerd" and that is as far as I can go. Finally, I tried connecting via FileZilla but that simply did not work. Any ideas on how I can reach the YAML file some other way?

Thank you again.

**Reply by EmptyWolf** [reply]
Hey!
Quick update, I ended up finding AdGuardHome.yaml (in a weird location). However, the file is read-only so I can't change any values.. :(
